src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -4,5 +4,13 @@
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    public static void main(String[] args) {
+        FigureSupplier figure = new FigureSupplier();
+        Figure [] array = {figure.getRandomFigure(),figure.getDefaultFigure(),
```

Comment:
You need to create an array of figures with a specific length. Then, use a loop to iterate through each element in the array. Inside the loop, check whether the current index is in the first or second half of the array. If it's in the first half, add a RandomFigure to the array, and if it's in the second half, add a DefaultFigure.